fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -3,6 +3,7 @@
 
 var test = require('tap').test
 var crypto = require('crypto')
+var URLSafeBase64 = require('urlsafe-base64')
```

Comment:
Why does this use a different module to the `base64url` one that's used in the auth-server PR?